### ISSUE TITLE
Keyboard navigation: opening a submenu on the strategy HUD doesn't show any subitem as active

### ISSUE THREAD

**Issue by chrishayesmu** [chrishayesmu/XCOM-LW-CommunityEdition#36]
When using the keyboard (and possibly controller, untested) to navigate the top level strategy HUD, if you open a submenu, no subitem will be highlighted. However, if you then press enter, the topmost item is activated, suggesting it was active the whole time. This item should be highlighted upon opening the menu.
